Vizzuality/marxan-cloud
# [MRXN23-264] Breadcrumbs

## Breadcrumbs

### Overview

New layout breadcrumbs component:

![Screenshot 2023-08-01 at 18 34 04](https://github.com/Vizzuality/marxan-cloud/assets/51995866/9ec2f56a-cc79-44b2-827f-fd4158bce62d)

### Designs

[Designs](https://www.figma.com/file/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=1453%3A14822&mode=dev)

### Testing instructions

Access dashboard, project and scenario through the app and check that the page you are on is blue in the breadcrumbs and also you can navigate with them.

### Feature relevant tickets

[JIRA task](https://vizzuality.atlassian.net/browse/MRXN23-264)
